### PR TITLE
Update key vault jca readme

### PR DESCRIPTION
**Pull request by chenrujun**
Update key vault jca readme

Related PR: https://github.com/Azure/azure-sdk-for-java/pull/19051

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultLoadStoreParameter.java [hunk 1]
```diff
@@ -10,6 +10,8 @@
  */
 public class KeyVaultLoadStoreParameter implements KeyStore.LoadStoreParameter {
 
+    private static final String DEFAULT_AAD_AUTHENTICATION_URL = "https://login.microsoftonline.com/";
```
Comment (lzc-1997-abel) [inline sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultLoadStoreParameter.java:13]: Should we also set the URL [here](https://github.com/chenrujun/azure-sdk-for-java/blob/2ad3072ff46f2163ddb878d2adfe909e20f9a3dc/sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultLoadStoreParameter.java#L62) as the default URL

**Comment by lzc-1997-abel** [comment]
LGTM

**Review by chenrujun — COMMENTED** [review]
LGTM.

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**